modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -36,7 +36,8 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * AWS: `LoadBalancerService` (with external scope)
 * Azure: `LoadBalancerService` (with external scope)
 * GCP: `LoadBalancerService` (with external scope)
-* Other: `HostNetwork`.
+* Bare metal: `NodePort`
```

Comment:
I'm not sure if `NodePort` is also interchangeable with HostNetwork`. Is it appropriate to align it with bare metal? This is subject to change.

Reply:
https://github.com/openshift/cluster-ingress-operator/blob/1f3fbf43b4de34f0185271f894372f55252bee43/pkg/operator/controller/ingress/controller.go#L302
Seems the default value for Bare metal is still `HostNetwork`.

Reply:
So we should add another section to tell how to change from default `HostNetwork` to `NodePortService`.